javascript/URDFLoader.js

```diff
@@ -403,23 +412,40 @@ class URDFLoader {
     }
 
     // Process the <link> nodes
-    _processLink(link, packages, path, loadMeshCb) {
+    _processLink(link, materials, packages, path, loadMeshCb) {
```

Comment:
Maybe a nit pick but can we change the `materials` argument here to `materialMap` so it's consistent?